### PR TITLE
create test environment scripts to support test and CI

### PR DESCRIPTION
**Pull request by tvieira**
This commit adds few scripts that are part of the CI automation #40 that we want
create for this project. The scripts are:

* testinstance - which is a python script to download ManageIQ docker image, start
and stop a ManageIQ instance. (requires Docker and python Docker library)

* change_configuration.js - PhantomJS script that will configure the local ManageIQ
instance to prepare it for testing purpose. For this commit only changing the switch
for git owner is set to allow us to upload our datastore. (requires PhantomJS)

* install_datastore.js - PhantomJS script that will import our datastore within the local
ManageIQ instance. (requires PhantomJS)

The file phantomjs-config.json is a simple configuration for PhantomJS to ignore
SSL errors (when a local ManageIQ container runs, its certificate is not valid. This
will cause the PhantomJS scripts to fail.

### CODE REVIEW

**Comment by tvieira** [comment]
I have to fix tox. PR is coming...

File: tox.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.0
-envlist = py36,py27,pep8,docs
```
Comment (ryankwilliams) [inline tox.ini]: Is there a reason `docs` is removed from the envlist?

**Review by ryankwilliams — APPROVED** [review]
LGTM

File: tox.ini (hunk 1 above)
Comment (tvieira) [reply tox.ini]: Oops, no. My bad. I will be putting it back shortly.

**Review by vi-patel — APPROVED** [review]
looks good.

**Pull request merged**